client/src/planwise/client/projects2/components/settings.cljs

```diff
@@ -392,13 +393,28 @@
                                                               " "
                                                               demand-unit])]) capacities))]))
 
+(def step->component
+  {"goal"      current-project-step-goal
+   "consumers" current-project-step-consumers
+   "providers" current-project-step-providers
+   "coverage"  current-project-step-coverage
+   "actions"   current-project-step-actions
+   "review"    current-project-step-review})
+
+(defn step->spec
+  [step]
+  (keyword "planwise.model.project" (str step "-step")))
```

Comment:
This is fine, but I would have left the specs a part of the `core/sections` definition. That way the spec is not necessarily fixed from the step name.